system-test/storage.js

```diff
@@ -696,6 +696,25 @@ describe('storage', function() {
     });
   });
 
+  describe('channels', function() {
```

Comment:
Well, for the API call to work, it would have to be able to ping the created server. Running within a localhost environment, that would be problematic. Slash face.